### DISCUSSION TITLE
<x-filament::...> documentation

### DISCUSSION THREAD

**Discussion by ChimitovZ** [filamentphp/filament#3976] [Help] [answered]
Hello everyone!

Can you tell me please, where i can find some documentation (or at least examples) for things like
```html
<x-filament::widget>
<x-filament::card>
<x-filament::card.heading>
<x-tables::pagination :paginator="what i should provide here?">
<x-tables::table>
<x-tables::row>
<x-tables::filters>
...
```
and another...

**For example**: how can i draw table (with pagination, sorting, and maybe searching) on custom widget?

Thank you!

**Comment by danharrin** [comment, marked as answer]
Those are not documented atm as they are internal, but in v3 I will solidify the API and document them

> how can i draw table (with pagination, sorting, and maybe searching) on custom widget?

We have a table widget for this

**Reply by ChimitovZ** [reply]
Oh, Thank you for fast reply!

**Comment by DreadfulCode** [comment]
I am also looking for a list these:

<x-filament::widget>
<x-filament::card>
<x-filament::card.heading>
<x-tables::pagination :paginator="what i should provide here?">
<x-tables::table>
<x-tables::row>
<x-tables::filters>

I don't see any documentation on this anywhere, and would like to compile such a list if possible.

**Comment by BrunoTrndd** [comment]
also would like to know if there is any documentation atm, cuse the answer was from 2022, by now it should've been documented, right?

**Reply by dissto** [reply]
There is for many of them

https://filamentphp.com/docs/3.x/support/blade-components/overview

**Comment by LukaszJaro** [comment]
I found this site which has a little more documentation for the blade components, like card for example https://docs.laravel-filament.cn/docs/support/blade-components/card

**Reply by danharrin** [reply]
You shouldn't use that Blade component, it's deprecated. It's just the same as `<x-filament::section>` without a heading. Hence why it is not documented on our site. I don't control that one.
